BLSQ/iaso
# IA-2910 IA-2911 hide planning and  entitie srelated mobile feature flags

Explain what problem this PR is resolving
- Hide Planning-related mobile feature flag if Planning module is not activated
- Hide Entities-related mobile feature flags if Entities module is not activated
Related JIRA tickets : [IA-2910](https://bluesquare.atlassian.net/browse/IA-2910) [IA-2911](https://bluesquare.atlassian.net/browse/IA-2911)
## Self proofreading checklist


## Doc
- **/iaso/docs/pages/dev/how_to/exclude_featureflag_related_module/exclude_featureflag_related_module.md**

## Changes
- Add get action method in api/featureflags
- Add FEATUREFLAGES_TO_EXCLUDE to link modules and featureflags to be excluded when modules are not activated

## How to test
- Create an account or edit the current one
- Remove the Planning and the entities modules
- Update or create a project
- Check if featureflags listed in jira tickets appear in dropdown list

## Print screen / video
[Screencast from 2024-05-07 13-41-13.webm](https://github.com/BLSQ/iaso/assets/19631540/abdcbe5f-501e-43ae-bfc9-b108e30f7d46)


[IA-2910]: https://bluesquare.atlassian.net/browse/IA-2910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-2911]: https://bluesquare.atlassian.net/browse/IA-2911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
